python/django/python-hello-world/src/helloapp/templates/homepage.html

```diff
@@ -0,0 +1,197 @@
+<!--
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
We should remove this - the samples are licensed as bsd0, not apache2

Reply:
Google typically uses the apache2 license.  Why are the samples BSD?